config/locales/application_form.yml

```diff
@@ -803,6 +803,33 @@ en:
               in_future: Enter a year before %{date}
             other_grade:
               blank: Enter your grade
+        candidate_interface/science_gcse_grade_form:
+          attributes:
+            single_award_grade:
+              blank: Enter your grade
+              invalid: Enter a real grade
+            double_award_grade:
+              blank: Enter your grade
+              invalid: Enter a real grade
+            grade:
+              blank: Enter your grade
+              invalid: Enter a real grade
+              too_long: Grade must be %{count} characters or fewer
+            biology_grade:
+              invalid: Enter a real grade
+              blank: Enter your grade
+            chemistry_grade:
+              invalid: Enter a real grade
+              blank: Enter your grade
+            physics_grade:
+              invalid: Enter a real grade
+              blank: Enter your grade
+            award_year:
+              blank: Enter the year you gained your qualification
+              invalid: Enter a real year
+              in_future: Enter a year before %{date}
+            other_grade:
+              blank: Enter your grade
```

Comment:
This file is very long. Would it be worth starting to split it out? Or are we intending to keep all the `application_form` strings in one place?

Reply:
@raamSoftwire https://trello.com/c/g7vXu7sI/367-low-localisations-should-match-structure-of-application

Reply:
@tvararu Was just going to enthusiastically comment on that card – until I realised that I wrote it! 😂